docs/mobile/deploys.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: Deploys en Mobile
+description:
+  '¿Como hacemos que otra persona pueda ver nuestras apps en los stores (Google Play / Apple Store)? <br />
+  ¿Que herramientas nos permiten publicarla, configurarla, entre otras necesidades? <br />
+  Tu elección va a depender de las necesidades del proyecto y las preferencias del equipo de desarrollo. <br />
+  Algunas de las herramientas más populares son:'
+
+contributors:
+  - github_username: 'jpaddeo'
+
+submenu:
+  - imgPlaceholder: 'Expo EAS'
+    titleCard: ''
+    links:
+      - type: 'Web'
+        url: 'https://expo.dev/eas'
+    cover:
+      src: '/mobile/expo.svg'
+      height: 80
+      width: 100
+    createdAt: '2024-01-30T00:00:00Z'
```

Comment:
Estas fechas se quitaron, podrás eliminarlas? createdAt y updatedAt

Reply:
Of cors! Perdón por la demorísima!